test/command/markdown-to-asciidoc-headings.md

```diff
@@ -0,0 +1,94 @@
+converted from: <https://github.com/bodiam/markdown-to-asciidoc/tree/9bd90bc405a7d25b03822cc91154bcb315ab39bf/src/test/resources/nl/jworks/markdown_to_asciidoc/headings.feature>
+
+Render a level 1 heading
+
+```
+% pandoc -t asciidoc
+# Title #
+^D
+= Title
+```
+
+
+Render a level 1 underscored heading
+
+```
+% pandoc -t asciidoc
+Title
+=====
+^D
+= Title
+```
+
+
+Render a level 2 heading
+
+```
+% pandoc -t asciidoc
+## Title
+^D
+== Title
+```
+
+
+Render a level 2 underscored heading
+
+```
+% pandoc -t asciidoc
+Title
+-----
+^D
+== Title
+```
+
+
+Render a level 3 heading
+
+```
+% pandoc -t asciidoc
+## Title
```

Comment:
shouldn't this read `### Title`?

Reply:
Maybe :) We should probably implement https://github.com/jgm/pandoc/issues/5615